.github/workflows/pr-checks.yaml

```diff
@@ -97,3 +97,52 @@ jobs:
       - name: Build image
         working-directory: ./pkg/adapter/${{ matrix.adapters }}
         run: make docker-build
+
+  chainsaw-tests:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout
+          uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+          # install tools
+        - name: Install helm
+          id: helm
+          uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
+          with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+          
+          # install go
+
+        - name: Install go
+          uses: actions/setup-go@v5
+          with:
+            go-version: '1.21.7' # The Go version to download (if necessary) and use.
+          
+
+        # install chainsaw
+
+        - name: Install chainsaw
+          uses: kyverno/action-install-chainsaw@8307cf9038333ce168dd4339ae24f9ab16ab7a00 # v0.1.4
+
+        # create kind cluster
+
+        - name: Create k8s Kind Cluster
+          uses: helm/kind-action@v1
+
+
+        - name: start server
+          run: |
+            make install
+            make run &
```

Comment:
Why do we need this?
Can't we use the following:
```shell
helm upgrade --install nimbus-operator . -n nimbus --create-namespace --set image.pullPolicy=Never
```
?

Reply:
It's showing the  error while deploying nimbus via locally built image.

Reply:
![image](https://github.com/5GSEC/nimbus/assets/82467006/a40af54d-b1cb-46e2-80ff-a6ed1b4e2a2e)